datadoc/backend/DatasetReader.py

```diff
@@ -72,25 +72,26 @@ def __init__(self, dataset):
     @staticmethod
     def for_file(dataset: str) -> TDatasetReader:
         """Factory method to return the correct subclass based on the given dataset file"""
-        SUPPORTED_FILE_TYPES = {
+        supported_file_types = {
             "parquet": DatasetReaderParquet,
```

Comment:
Perhaps the DatasetReaders should be renamed to DatasetParser, so that we can have different Readers per filesystem (e.g. GCS)?

Reply:
Good suggestion! Yes it does look like we'll need two abstraction levels here, one for file type and one for storage medium. Is it alright with you if we make that change when we integrate with GCS?

Reply:
Noted under the GCS integration issue, we'll make the change there: https://statistics-norway.atlassian.net/browse/STAT-16?focusedCommentId=22689